src/server/test/web/meters.js

```diff
@@ -22,10 +22,18 @@ mocha.describe('meters API', () => {
 
 	mocha.it('returns all visible meters', async () => {
 		const conn = testDB.getConnection();
-		await new Meter(undefined, 'Meter 1', '1.1.1.1', true, true, Meter.type.MAMAC, 'TZ1', gps).insert(conn);
-		await new Meter(undefined, 'Meter 2', '1.1.1.1', true, true, Meter.type.MAMAC, 'TZ2', gps).insert(conn);
-		await new Meter(undefined, 'Meter 3', '1.1.1.1', true, true, Meter.type.MAMAC, 'TZ3', gps).insert(conn);
-		await new Meter(undefined, 'Not Visible', '1.1.1.1', true, false, Meter.type.MAMAC, 'TZ4', gps).insert(conn);
+		await new Meter(undefined, 'Meter 1', '1.1.1.1', true, true, Meter.type.MAMAC, 'TZ1', gps, 
+		'IDENTIFIED', 'Notes', 33.5, true, true, '05:05:09', '09:00:01', true, '00:00:00','00:00:00', 25.5, 
+		'0011-05-022 : 23:59:59', '2020-07-02 : 01:00:10').insert(conn);
+		await new Meter(undefined, 'Meter 2', '1.1.1.1', true, true, Meter.type.MAMAC, 'TZ2', gps, 
+		'Identified 1' ,'Notes', 35.0, true, true, '01:01:25' , '00:00:00', true, '05:00:00','00:00:00', 1.5,
+		'0011-05-22 : 23:59:59', '2020-07-02 : 01:00:10').insert(conn);
+		await new Meter(undefined, 'Meter 3', '1.1.1.1', true, true, Meter.type.MAMAC, 'TZ3', gps,
+		'Identified 2', 'Notes', 35.0, true, true, '01:01:25' , '00:00:00', true, '05:00:00','00:00:00', 1.5, 
+		'0011-05-022 : 23:59:59', '2020-07-02 : 01:00:10').insert(conn);
+		await new Meter(undefined, 'Not Visible', '1.1.1.1', true, false, Meter.type.MAMAC, 'TZ4', gps, 
+		'Identified 3' ,'Notes', 35.0, true, true, '01:01:25' , '00:00:00', true, '05:00:00','00:00:00', 1.5, 
+		'0011-05-022 : 23:59:59', '2020-07-02 : 01:00:10').insert(conn);
```

Comment:
I cannot comment on lines you did not change so I'm putting this comment here. I think you really should be testing the new meter values. You added them to the meters you created but you never verify they have the correct values as the previous values did. I'm also uncertain why some meter have the new values and others were not changed.